Api/src/controllers/officeController.js

```diff
@@ -0,0 +1,78 @@
+// import {
+//   v4 as uuidv4
+// } from 'uuid';
+
+import data from "../data.json" assert { type: "json" };
+
+let offices = data.Offices;
+
+export const getOffices = async (req, res) => {
+  res.statusCode = 200;
+  res.setHeader('Content-Type', 'application/json');
+  res.send((data.Offices));
+
+  // Leaving this for reference 
+  // console.log(Object.keys(data));
+};
+
+export const createOffice = (req, res, err) => {
+  let office = req.body;
+
+  // For creating unique office IDs
+  // const officeId = uuidv4();
+
+  const officeWithId = {
+    ...office
+  };
+
+  offices.push(officeWithId);
+
+  res.send({
+    status: 200,
+    data: officeWithId,
+  });
+};
+
+export const getOffice = (req, res) => {
+  const {
+    id
+  } = req.params;
+
+  const foundOffice = data.Offices.find((office) => office.id === parseInt(id));
+
+  if (!foundOffice) {
+    res.statusCode = 400
+    res.send("Office not found.");
+  };
+
+  res.send(foundOffice);
```

Comment:
We're still failing the expectations of the API Endpoint Specification. This is not what should be returned for `GET /offices/:id`

Reply:
done.